spec/models/manageiq/providers/base_manager/test_persister.rb

```diff
@@ -0,0 +1,8 @@
+module ManageIQ::Providers
```

Comment:
I think this should live in `spec/support/models` instead, since it's really a supporting thing...otherwise it looks like an actual test file that accidentally didn't end in `_spec`.

(There may also actually be a conflict naming it `test_` with some other test suites, not that we use them, but we used to have things like the MiqDisk tests that were all named `test_`)

Reply:
We already have a `spec/support/ems_refresh_helper` so what about `spec/support/ems_refresh_helper/test_persister` ?